From 531b6f6c429678f69f2c90280a1e706eed6dcfbc Mon Sep 17 00:00:00 2001
From: Igor Murzov <igor@gplsoft.org>
Date: Sun, 18 Apr 2010 22:56:56 +0400
Subject: [PATCH 4/6] produce more compliant slack-desc

slackware uses slightly different format for slack-desc files
from that which is used by rpm2tgz;
also remove all -l 70 options as it is only usable with l command
---
 rpm2tgz |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/rpm2tgz b/rpm2tgz
index 4818b74..cc25407 100755
--- a/rpm2tgz
+++ b/rpm2tgz
@@ -204,8 +204,7 @@ for i in $* ; do
   if [ "$DESC" = "true" ]; then
     get_meta_data $i
     mkdir -p $TMPDIR/install
-    rpm -qp --qf %{SUMMARY} $i | sed -l 70 -r "s/^(.*)/$PRGNAM: $PRGNAM - \1\n/" > $TMPDIR/install/slack-desc
-    rpm -qp --qf %{DESCRIPTION} $i | sed -l 70 -r "s/^/$PRGNAM: /" >> $TMPDIR/install/slack-desc
+    rpm -qp --qf '%{NAME} (%{SUMMARY})\n\n%{DESCRIPTION}\n\n  %{URL}\n' $i | head -n 11 | sed -r "s/^/$PRGNAM: /" > $TMPDIR/install/slack-desc
   fi
 
   # If this program was called as "rpm2targz", then repack as a plain
-- 
1.7.0.2

